website/docs/components/form/file-input/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,13 @@
+## Anatomy
+
+![Anatomy of File Input](/assets/components/form/file-input/file-input-anatomy.png)
+
+
+| Element           | Usage                                          |
+|-------------------|------------------------------------------------|
+| Label             | Required                                       |
+| Indicator         | Optional                                       |
+| Helper text       | Optional                                       |
+| Button            | Required                                       |
+| File status       | Required                                       |
```

Comment:
I'm not sure if marking this as "required" is actually true, since it's just generated by the browser. Technically I guess it is since the browser is injecting it, but the consumer doesn't have control over it, so it's not a required attribute on their end.